pylint_sqlalchemy/orm.py

```diff
@@ -0,0 +1,24 @@
+"""Pylint sqlachemy hack module."""
+
+
+class Session():
```

Comment:
Can we derive from `object` here while we wait for Python 2.7 to fully sunset?

```suggestion
class Session(object):
```